source/documentation/deploying_services/index.md

```diff
@@ -737,22 +737,22 @@ where `SERVICE_NAME` is a unique descriptive name for this service instance, and
 cf update-service my-ms-service -p medium-ha-5.7
 ```
 
-The plan upgrade will begin immediately and will complete within about an hour. You can check the status of the change by running the `cf services` command.
+The plan upgrade will start immediately and finish within an hour. You can check the status of the upgrade by running `cf services`.
 
-You can also [queue a plan upgrade](/deploying_services.html#queue-a-plan-migration-mysql) to happen during a maintenance window to minimise service interruption.
+You can also [queue a plan upgrade](/deploying_services.html#queue-a-plan-migration-postgresql) to happen during a maintenance window to minimise service interruption.
```

Comment:
I don't think you meant to change this link... This is in the mysql section.

Reply:
agree